common/test/acceptance/tests/lms/test_lms_courseware.py

```diff
@@ -441,7 +440,7 @@ class CoursewareMultipleVerticalsTest(CoursewareMultipleVerticalsTestBase):
     Test courseware with multiple verticals
     """
 
-    @skip('Disable temporarily to get course bookmarks out')
+    @flaky  # PLAT-1198; should be fixed, but verify that failures stop before removing
```

Comment:
Thanks for the tip!  Since I've already created the PR (and actually have a clear run of tests), I think I'm going to go ahead and merge it with the flaky decorator intact for now.  Also, it looks like Sandy tried the 100 runs trick earlier in this bug's lifecycle and wasn't able to reproduce the recurring Jenkins failure, so I'm not sure I'd trust the results on this one.